lib/miq_automation_engine/service_models/miq_ae_service_container_project.rb

```diff
@@ -0,0 +1,18 @@
+module MiqAeMethodService
+  class MiqAeServiceContainerProject < MiqAeServiceModelBase
+    expose :container_groups,       :association => true
+    expose :container_routes,       :association => true
+    expose :container_replicators,  :association => true
+    expose :container_services,     :association => true
+    expose :container_definitions,  :association => true
+    expose :container_nodes,        :association => true
+    expose :container_quotas,       :association => true
+    expose :container_quota_items,  :association => true
+    expose :container_limits,       :association => true
+    expose :container_limit_items,  :association => true
+    expose :container_builds,       :association => true
+    expose :metrics,                :association => true
+    expose :metric_rollups,         :association => true
+    expose :vim_performance_states, :association => true

```

Comment:
@gmcculloug preferably, yes...I'll create the necessary service models. With regards metrics, no particular use case - I'll remove.